localstripe/resources.py

```diff
@@ -161,11 +161,11 @@ def _api_delete(cls, id):
         return {"deleted": True, "id": id}
 
     @classmethod
-    def _api_list_all(cls, url, limit=None, **kwargs):
+    def _api_list_all(cls, url, limit=None, starting_after=None, **kwargs):
         if kwargs:
             raise UserError(400, 'Unexpected ' + ', '.join(kwargs.keys()))
 
-        li = List(url, limit=limit)
+        li = List(url, limit=limit, starting_after=starting_after)
```

Comment:
There are other `_api_list_all()` functions where the same needs to be done (in `Charge`, `Invoice`, `InvoiceItem`, `PaymentMethod`, `Refund` and `Subscription`).
Can you add them there too? You just need to take the argument and pass it.